store/rootmulti/store.go

```diff
@@ -312,6 +316,23 @@ func (rs *Store) TracingEnabled() bool {
 	return rs.traceWriter != nil
 }
 
+// SetListeners sets the listeners for a specific KVStore
+func (rs *Store) SetListeners(key types.StoreKey, listeners []types.WriteListener) {
```

Comment:
`SetListeners` reads like we're overwriting existing listeners for a particular store key, not appending listeners. Should we rename to `AddListeners`, wdyt?

Reply:
Yeah, that makes sense to me!